src/metapath-results.js

```diff
@@ -133,7 +136,7 @@ class TableFull extends Component {
   // checks whether all metapaths besides the specified are unchecked
   allOthersUnchecked(id) {
     for (const metapath of this.props.metapaths) {
-      if (id !== metapath.id && metapath.checked)
+      if (id !== metapath.metapath_id && metapath.checked)
```

Comment:
`metapath_id` is the orientation in the database table. Sometimes it will be reversed compared to the display... so not sure if it would make more sense to use `metapath.abbreviation`.

Reply:
This code is just to identify which is the right row of the table to access/change. So it really doesn't matter which I use, as long as the property is unique for each row.

In addition, in the refactor, it actually compares the whole object, which is more robust, so this will be irrelevant.